### DISCUSSION TITLE
because the character at index 19 has a value of 8230 which is greater than 255.

### DISCUSSION THREAD

**Discussion by code-andy-stackbaker** [clerk/javascript#4102] [Support request - DO NOT OPEN HERE]
Does anybody have faced this issue before? I'm using clerk latest version.

Cannot convert argument to a ByteString because the character at index 19 has a value of 8230 which is greater than 255.

		"@clerk/nextjs": "^5.4.1",
		"@clerk/themes": "^2.1.12",
		"next": "14.2.5",

**Comment by jescalan** [comment]
please open this issue with [clerk support](https://clerk.com/contact/support) instead!
